src/Module/Register.php

```diff
@@ -380,11 +380,11 @@ protected function post(array $request = [])
 					'type'         => Model\Notification\Type::SYSTEM,
 					'event'        => 'SYSTEM_REGISTER_REQUEST',
 					'uid'          => $admin['uid'],
-					'link'         => $base_url . '/admin/users/',
+					'link'         => DI::baseUrl()->get(true) . '/admin/users/',
```

Comment:
why is this replaced? Doesn't `$base_url` already contain the base url?

Reply:
We have this legacy behavior where the regular pages can be accessed with HTTP (the base URL of the node), but the admin pages are only accessed with HTTPS (self-signed certificate). In this case, `DI::baseUrl()->get()` (the value of `$baseurl`) will return the HTTP version, and `DI::baseUrl()->get(true)` will return the HTTPS version.

It's a very niche use case and we might want to remove it, but in the mean time it should be applied consistently.